src/cirrus/plugins/builders/conda_conda.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+import os
+from cirrus.builder_plugin import Builder
+from cirrus.logger import get_logger
+from cirrus.invoke_helpers import local
+
+
+LOGGER = get_logger()
+
+
+class Conda(Builder):
```

Comment:
I think I need to rename the module to just conda... 
The class name ends up being the plugin cli name as well, and while CondaConda sounds like some sort of jungle disease/fruity cocktail, it isnt really well suited to a pure conda plugin name :)